### ISSUE TITLE
Determine which backend is currently used

### ISSUE THREAD

**Issue by athenaleong** [LouisBrunner/dnd-multi-backend#104]
Hey! Thanks for the great package. 

Is there a way to figure out which backend is being used so I can generate different previews?

**Comment by LouisBrunner** [comment]
Hello @athenaleong,

Would you mean as a return value of `usePreview`?

**Comment by vezaynk** [comment]
@LouisBrunner I'd like to be able to simply log which back-end is currently active. My dnd is not working on iOS and I'm struggling to debug it.

**Comment by LouisBrunner** [comment]
> @LouisBrunner I'd like to be able to simply log which back-end is currently active. My dnd is not working on iOS and I'm struggling to debug it.

Fair enough. How are you currently using this package? I mean specifically which hooks, class, etc.
